spinalcordtoolbox/scripts/sct_process_segmentation.py

```diff
@@ -184,6 +188,12 @@ def get_parser():
         metavar=Metavar.folder,
         action=ActionCreateFolder,
         help="The path where the quality control generated content will be saved."
+             "QC report is only availbale for PMJ-based CSA."
```

Comment:
```suggestion
             " The QC report is only available for PMJ-based CSA (with flag '-pmj')."
```
A space was needed at the beginning of the line.